components/sponsors/SponsorCard.vue

```diff
@@ -36,9 +36,9 @@ export default {
 <style lang="postcss" scoped>
 .sponsorCard {
     @apply relative flex flex-col justify-center items-center bg-white rounded-2xl;
-    width: 40%;
+    width: 25%;
     @media (min-width: 767px) {
-        @apply w-40;
+        @apply w-1/5;
```

Comment:
I want 4 sponsorship cards in a row, but have no idea why we need to specify `w-1/5` instead of `w-1/4`